libgalois/include/katana/GraphHelpers.h

```diff
@@ -245,15 +245,29 @@ divideNodesBinarySearch(
                                   edgePrefixSum, nodesUpper - 1 + nodeOffset) -
                                   edgeOffset;
 
-  // katana::gDebug("Unit ", id, " nodes ", nodesLower, " to ",
-  //               nodesUpper, " edges ", edgesLower, " ",
-  //               edgesUpper);
+  katana::gDebug(
+      "Unit ", id, " nodes ", nodesLower, " to ", nodesUpper, " edges ",
+      edgesLower, " ", edgesUpper);
 
```

Comment:
you should probably recomment this out or remove it entirely (this function is called quite a bit in distributed execution IIRC, and it will blow up debug output)

Reply:
I think I commented this out to see if it would compile. I haven't seen `katana::gDebug` before. I guess it does! I will probably just remove it.